packages/compartment-mapper/src/import-hook.js

```diff
@@ -125,6 +126,9 @@ export const makeImportHookMaker = (
       // The `moduleMapHook` captures all third-party dependencies.
       if (moduleSpecifier !== '.' && !moduleSpecifier.startsWith('./')) {
         if (has(exitModules, moduleSpecifier)) {
+          gatekeepModuleAccess(moduleSpecifier, compartmentDescriptor.policy, {
+            exit: true,
+          });
```

Comment:
I'm not sure about this. I don't see how execution would get to here. @kriskowal ?

Reply:
There are no tests covering it. 
How about we keep it as is for now and I'll create a new PR replacing this with an error that this should never happen?